hera_corr_cm/hera_corr_cm.py

```diff
@@ -469,12 +473,14 @@ def get_snaprf_status(self, numch=6):
                         try:
                             rf_status[rfch][key] = cfunc(stats[host][ckey], carg).astype(ccst)
                         except Exception as e:
-                            rf_status[rfch][key] = "Exception: {}".format(str(e))
+                            # rf_status[rfch][key] = "Exception: {}".format(str(e))
+                            rf_status[rfch][key] = None
                     else:
                         try:
                             rf_status[rfch][key] = cfunc(stats[host][ckey].decode())
                         except Exception as e:
-                            rf_status[rfch][key] = "Exception: {}".format(str(e))
+                            # rf_status[rfch][key] = "Exception: {}".format(str(e))
```

Comment:
remove this

Reply:
I've removed all of the commented lines.